apps/form-app/src/app/containers/Form.tsx

```diff
@@ -77,6 +77,13 @@ const FormComponent: FunctionComponent<FormProps> = ({ className }) => {
                   showSubmit={showSubmit}
                   saving={busy.saving}
                   onChange={({ data, errors }) => {
+                    if (
+                      errors[0]?.message === 'should be equal to one of the allowed values' &&
+                      errors[0]?.schemaPath.includes('enum')
+                    ) {
+                      errors = null;
+                    }
+
```

Comment:
This seems a bit dodgy.  Why is this code trying to filter out very specific error messages?  And even if this is a valid thing to do, the code would be much cleaner (and easier to digest) if you made this into a function, e.g.
errors = filterIgnoredErrors(errors)

Reply:
I could instead change the ui schema before it gets validated. not sure which is better. maybe that's better

Reply:
Generally speaking, depending on the text of an error message eventually leads to unexpected and hard to detect runtime errors.  If you change the UI schema before had, can you avoid this situation?

Reply:
yes, I already kind of do a similar thing in this code already, so it wouldn't be hard